stakingModel/contracts/InterestDistribution.sol

```diff
@@ -180,7 +172,8 @@ library InterestDistribution {
       * 
       * Formula:
       * AvgYieldRatePerToken = [(AvgYieldRatePerToken(P) x TotalStaked) + (AccumulatedYieldPerToken(P) x Staking x (1-%Donation))]/TotalStaked
-      * 
+      *
+      * @param _stakerData                  Staker's Data
```

Comment:
the formula you implemented is incorrect

Reply:
I implemented same formula that is mentioned in [here](https://docs.google.com/document/d/18RG1Mhips1qCT0ya9QKbTOHKRDNJNQ8YxZrxpMQqe98/edit)

please let me know if the doc is not updated. @sirpy